test/samplers.jl

```diff
@@ -82,7 +82,7 @@ end
 
     s1 = NStepBatchSampler(n=n_horizon, γ=γ, stack_size=n_stack, batch_size=batch_size)
 
-    xs = RLTrajectories.sample(s1, t1)
+    xs = RLTrajectories.StatsBase.sample(s1, t1)
```

Comment:
Can't we add a method to StatsBase.sample for our types? That way we would only need to do `sample(s1,t1)`?

Reply:
`sample` is not exported by RLTrajectories because it is not intended to be part of its api (it bypasses the controller). 

If you don't like having to do `RLTrajectory.StatsBase.sample` we can import sample into RLTrajectories. That way the namespace would be unchanged compared to before (that is, we would still have to do `RLTrajectories.sample`). 

If you would like to be able to do `sample` then we can reexport the name. Which one do you prefer ?